shell.nix

```diff
@@ -58,13 +65,15 @@ in haskell.packages.shellFor {
   # we have a local passwords store that we use for deployments etc.
   PASSWORD_STORE_DIR = toString ./. + "/secrets";
 
-  shellHook = 
+  shellHook = ''
+    ${pre-commit-check.shellHook}
+  ''
     # Work around https://github.com/NixOS/nix/issues/3345, which makes
     # tests etc. run single-threaded in a nix-shell.
     # Sets the affinity to cores 0-1000 for $$ (current PID in bash)
     # Only necessary for linux - darwin doesn't even expose thread
     # affinity APIs!
-    pkgs.lib.optionalString pkgs.stdenv.isLinux ''
-      taskset -pc 0-1000 $$ 
+   + pkgs.lib.optionalString pkgs.stdenv.isLinux ''
+      ${pkgs.utillinux}/bin/taskset -pc 0-1000 $$
```

Comment:
Looks like a good improvement, but shouldn't it be a separate commit/PR?

Reply:
This was necessary because without it the `eval "$shellHook"` that i suggest people should add to their `.envrc` is going to fail.

Reply:
Ah, I see. Okay. 👍